ext/cgi/escape/escape.c

```diff
@@ -0,0 +1,94 @@
+#include "ruby.h"
+#include "ruby/encoding.h"
+
+static VALUE rb_cCGI, rb_mUtil, rb_mEscape;
+
+static VALUE
+html_escaped_cat(VALUE str, char c)
```

Comment:
Thank you. Fixed in https://github.com/k0kubun/ruby/commit/39cdd833d20b6e644c941486abb79a061c9330da.